cartography/intel/aws/lambda_function.py

```diff
@@ -52,7 +58,7 @@ def load_lambda_functions(
     lambda.region = lf.region,
     lambda.modifieddate = lf.LastModified,
     lambda.runtime = lf.Runtime,
-    lambda.isPublicFacing = lf.isPublicFacing,
```

Comment:
since we are removing this attribute, we should check it's impact. If it's used somewhere else. like in inventory project, we should update that to use the new attribute.